include/swift/AST/DiagnosticsSema.def

```diff
@@ -864,6 +869,9 @@ WARNING(module_not_compiled_with_library_evolution,none,
         "using it means binary compatibility for %1 can't be guaranteed",
         (Identifier, Identifier))
 
+REMARK(cross_import_added,none,
+       "Import of %0 and %1 triggered a cross-import of %2",
```

Comment:
Our diagnostics generally start with a lowercase letter, so this should start with `"import of...`

Reply:
(If you haven't read it before, there's a file in the docs folder about our diagnostic style: https://github.com/apple/swift/blob/master/docs/Diagnostics.md)

Reply:
👍Thanks for the link!